planning/behavior_path_planner/config/scene_module_manager.param.yaml

```diff
@@ -5,69 +5,79 @@
   ros__parameters:
     external_request_lane_change_left:
       enable_module: false
+      enable_auto_mode: false
```

Comment:
The name `auto_mode` looks ambiguous for users. It should be more specific, for example, `enable_rtc`. (enable_rtc itself is not clear to understand the behavior. The description should be on the readme.)